templates/base.html

```diff
@@ -17,7 +17,6 @@
         {% else %}
         <meta name="author" content="{{ AUTHOR }}" />
         {% endif %}
-
```

Comment:
This file should not be in the PR at all. 😄

Reply:
Done!